### PR TITLE
Implementation of LOOCV Ridge Regression method

### PR DESCRIPTION
**Pull request by narutak**
Thank you for all the work @atztogo. I have implemented the computation of the force constants using Leave-One-Out Cross Validation (LOOCV) Ridge Regression to symfc-alm.

- Implementation of LOOCV Ridge Regression: I have implemented a RidgeRegression class, which you can find in src/symfc_alm/symfc_alm.py.
- Data standardization function: Additionally, within the same file, src/symfc_alm/symfc_alm.py, I have also incorporated a function for standardizing input data, which will aid in the performance of our Ridge Regression model.

In order to verify the functionality and reliability of the new implementations, I've written two sets of tests:
- RidgeRegression Class Tests: You can find the tests for the RidgeRegression class in tests/test_ridge.py.
- RidgeRegression Run Tests: I have also written tests to ensure that the RidgeRegression model runs smoothly. They are available in tests/test_symfc-alm.py.

Please feel free to review the changes and leave any feedback or comments. I look forward to your review of the pull request.

### CODE REVIEW

File: tests/test_ridge.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+
+@pytest.fixture
+def regression():
+    """Get RidgeRegression class."""
+    return RidgeRegression()
+
+
+@pytest.fixture
+def data(si_111_dataset: DispForceDataset, si_111_structure: CellDataset):
```
Comment (atztogo) [inline tests/test_ridge.py]: `si_111_dataset` is a full-rank dataset. So tests with another rank deficient data are expected.

File: tests/test_ridge.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+import numpy as np
+import pytest
+
+from symfc_alm import CellDataset, DispForceDataset, SymfcAlm
+from symfc_alm.ridge import RidgeRegression, standardize_data
+
+
+@pytest.fixture
+def regression():
```
Comment (atztogo) [inline tests/test_ridge.py]: Is this fixture necessary?

File: tests/test_ridge.py (hunk 1 above)
Comment (atztogo) [reply tests/test_ridge.py]: This fixture can be placed in `conftest.py` with a longer variable name than `data`.

**Review by atztogo — CHANGES_REQUESTED** [review]

File: tests/test_ridge.py (hunk 1 above)
Comment (narutak) [reply tests/test_ridge.py]: I have made the corrections as requested. Please review tests.

File: tests/test_ridge.py (hunk 2 above)
Comment (narutak) [reply tests/test_ridge.py]: I have removed it and fixed the code to create a RidgeRegression class for each function. Please review it.

**Comment by narutak** [comment]
I've added the debias option to RidgeRegression._fit() of src/symfc_alm/ridge.py.

File: tests/test_ridge.py (hunk 1 above)
Comment (atztogo) [reply tests/test_ridge.py]: Thanks @narutak. Looks good.

**Comment by narutak** [comment]
Please review the code and if there are no issues, I request you to merge it.

**Review by atztogo — APPROVED** [review]

**Comment by atztogo** [comment]
Thanks @narutak. I merged manually excluding the commit adding `.vscode/settings.json`.

**Pull request closed without merging**